examples/publisher_example.rs

```diff
@@ -0,0 +1,61 @@
+/********************************************************************************
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+use std::{str::FromStr, time::SystemTime};
+
+use up_client_mqtt5_rust::{MqttConfig, UPClientMqtt, UPClientMqttType};
+use up_rust::{UMessageBuilder, UPayloadFormat, UStatus, UTransport, UUri, UUID};
+
+#[tokio::main]
```

Comment:
We chatted about this on other comments on the PR, but it's kinda odd to see `#[async_std::test]` over top the unit tests and `#[tokio::main]` here.

Reply:
updated to remove async_std